src/neo/Network/P2P/TaskManager.cs

```diff
@@ -24,11 +24,13 @@ private class Timer { }
 
         private static readonly TimeSpan TimerInterval = TimeSpan.FromSeconds(30);
         private static readonly TimeSpan TaskTimeout = TimeSpan.FromMinutes(1);
+        private static readonly UInt256 HeaderTaskHash = UInt256.Zero;
+        internal static readonly UInt256 MemPoolTaskHash = UInt256.Parse("0x0000000000000000000000000000000000000000000000000000000000000001");
```

Comment:
Can it be `private`?

Reply:
It's used here

https://github.com/neo-project/neo/pull/1544/files#diff-7ce4a1b436634bf191dd9a82ed714ff1R34